### ISSUE TITLE
unable to display the map

### ISSUE THREAD

**Issue by baboulinet382** [MatthewClarkMay/geoip-attack-map#24]
![capture3](https://user-images.githubusercontent.com/36900326/37486914-4a80a1c0-2866-11e8-9430-6585b70fe6b2.PNG)

Hey guys,

When I install geoip-map-attack everything works fine except the world map display.

Can you help Me Please.

Thanks

**Comment by gerszo88** [comment]
Hi! You need to create a MapBox Account and then put de map code into the code.

**Comment by soulserkut** [comment]
edit: changing my answer 

https://www.mapbox.com/developer-network/

I don't know about the paid thing. It is pay as you go but I did not have to give up any payment information before getting an API token through the dev network link.

**Comment by neertel** [comment]
Create a MapBox Account and then copy Default public token
and paste under /AttackMapServer/static$ vi map.js   
L.mapbox.accessToken = "pk.1MCJ9.OLZ1qvbmkiLiY2pxd213b2eyh0MTd2CJhIjoeTQybZ4dDYiaZA8HdzVhdT1NSm4HPC2w";

Restart the Application server 
and it should work

**Comment by rocheston** [comment]
I can't get the map to display as well. blank. I configured the API key and maps style, still blank.

**Comment by thecowmilk** [comment]
Yes also me when I update the key it wont show the map. Any help?

**Comment by catalins95** [comment]
In index.html change this
mapbox js and css versions from 2.3.0 to 3.3.1(latest) and it will load your world-map

**Comment by markchua88** [comment]
i changed mapbox js and css versions from 2.3.0 to 3.3.1(latest) but still not loading the map. Can some one help?
